src/scrapers/hapoalim.js

```diff
@@ -42,8 +51,16 @@ async function fetchAccountData(page, options) {
   for (let accountIndex = 0; accountIndex < accountsInfo.length; accountIndex += 1) {
     const accountNumber = `${accountsInfo[accountIndex].bankNumber}-${accountsInfo[accountIndex].branchNumber}-${accountsInfo[accountIndex].accountNumber}`;
 
-    const txnsUrl = `${apiSiteUrl}/current-account/transactions?accountId=${accountNumber}&numItemsPerPage=150&retrievalEndDate=${endDateStr}&retrievalStartDate=${startDateStr}&sortCode=1`;
+    const balanceCreditUrl = `${apiSiteUrl}/current-account/composite/balanceAndCreditLimit?accountId=${accountNumber}&view=details`;
+    let summary;
+    try {
+      const balanceCreditResult = await fetchGetWithinPage(page, balanceCreditUrl);
+      summary = fetchSummary(balanceCreditResult);
+    } catch (e) {
```

Comment:
@chilikla why are you swallowing this exception?
If this call fails, the exception should probably propagate all the way up.

Reply:
@eshaham you think if there is no summary property, this account shouldn't be submitted at all?

Reply:
@chilikla we don't expect this call to fail on a regular basis, and once account summary will be something the users of this module expect, the scraper should crash in case it cannot get this expected data.

Reply:
@eshaham understood.

Reply:
@chilikla saw what you did in 79bef319cd557a91e9b3e5b3987729d86ae13076.
What I actually meant was, get rid of the `try..catch` clause altogether - you're still swallowing the exception ATM.